third_party/sycl/crosstool/CROSSTOOL.tpl

```diff
@@ -192,9 +191,9 @@ toolchain {
   unfiltered_cxx_flag: "-D__TIME__=\"redacted\""
   unfiltered_cxx_flag: "-no-canonical-prefixes"
 
-  compiler_flag: "-U_FORTIFY_SOURCE"
-  compiler_flag: "-D_FORTIFY_SOURCE=1"
-  compiler_flag: "-fstack-protector"
+  compiler_flag: "-DDISABLE_SKINNY=1"
```

Comment:
We should add this the the bzl_configure, as we don't always want to disable this. Ideally the crosstool shouldn't change between branches.